user-ops-indexer/user-ops-indexer-logic/src/repository/paymaster.rs

```diff
@@ -0,0 +1,117 @@
+use ethers::prelude::Address;
+use sea_orm::{
+    prelude::Expr, sea_query::IntoCondition, ColumnTrait, DatabaseConnection, EntityTrait,
+    FromQueryResult, IntoSimpleExpr, JoinType, QueryFilter, QueryOrder, QuerySelect,
+};
+
+use entity::user_operations::{Column, Entity};
+
```

Comment:
nit: if you remove new lines between imports they will be sorted using cargo clippy